src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { NavLink, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { TableTitles } from '../../types/TableTitles';
+import { sortPeople } from '../../helpers/sortPeople';
+import { useChangeSearchParams } from '../../helpers/useChangeSearchParams';
+
+type PersonListProps = {
+  people: Person[];
+};
+
+export const PersonList: React.FC<PersonListProps> = ({ people }) => {
+  const { name } = useParams();
+  const [searchParams] = useSearchParams();
+  const [setSearhParams] = useChangeSearchParams();
+
+  const column = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+  const male = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const query = searchParams.get('query') || '';
+
+  const handleSort = (prop: TableTitles) => {
+    setSearhParams(prop);
+  };
+
+  const arrowDirection = (prop: TableTitles) => {
+    if (prop === column && !order) {
+      return (
+        <span className="icon">
+          <i className="fas fa-sort-up" />
+        </span>
+      );
+    }
+
+    if (prop === column && order) {
+      return (
+        <span className="icon">
+          <i className="fas fa-sort-down" />
+        </span>
+      );
+    }
+
+    return (
+      <span className="icon">
+        <i className="fas fa-sort" />
+      </span>
+    );
```

Comment:
You can optimize this element duplication. 
Try to write func which returns your ready class ( vector). 
Then return the element with this class. 

```suggestion
    return (
      <span className="icon">
        <i className=`fas ${vector}` />
      </span>
    );
```